src/test/ActiveFixture.php

```diff
@@ -63,7 +63,8 @@ public function load(): void
                 throw new InvalidArgumentException('Unable to save fixture data');
             }
 
-            $this->ids[] = $arInstance->id;
+            $this->data[$key] = array_merge($row, $arInstance->toArray());
```

Comment:
Should this be just `$this->data[$key] = $arInstance;` ?

Reply:
Yes probably, I think I got a bit tunnel-visioned trying to keep it close to the base method in Yii's `ActiveFixture class: https://github.com/yiisoft/yii2/blob/master/framework/test/ActiveFixture.php#L86